cypress/integration/gov-create-opp.spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+
+describe('As a user authenticated via IDIR', function() {
+    beforeEach(function() {
+        // clean up db
+        cy.exec('dropdb -f digitalmarketplace')
+        // set up db
+        cy.exec('createdb digitalmarketplace')
+        cy.exec('npm run migrations:latest;')
+        cy.exec('psql digitalmarketplace -f ./cypress/fixtures/users.sql')
+        cy.visit('auth/createsession')
+        Cypress.Cookies.preserveOnce("sid")
+    })
+
+    it('creates and saves a draft of a new CWU opportunity', function() {
+
+        cy.visit("/opportunities/create")
+        cy.contains('Get Started').first().click()
+
+        // 1. Overview tab
+        cy.get('#cwu-opportunity-title').type('Cypress Opp')
```

Comment:
I know [IDs aren't the best selectors](https://docs.cypress.io/guides/references/best-practices#:~:text=Selector,Notes), but since we're considering a refactor, I didn't want to add `data-cy` everywhere.

Reply:
The recommendations are opinionated. I like https://testing-library.com/ 's philosophy of only relying on things a user can interact with for testing. There's actually a (cypress-testing-library)[https://testing-library.com/docs/cypress-testing-library/intro]! Haven't tried it yet, but this could allow you to write
```js
cy.findByLabelText(/opportunity title/i).type('Cypress Opp')
```

Reply:
Cool, thanks! I'll play with it in future tests

Reply:
Feature ticket for adding [testing-library](https://testing-library.com/docs/cypress-testing-library/intro) support? I'll second the [opinionated Guiding Principles](https://testing-library.com/docs/guiding-principles) of that project.

> write tests that closely resemble how your web pages are used

Reply:
I already installed it as part of this PR